src/components/TheComments.vue

```diff
@@ -24,20 +27,35 @@
             rounded
           >
             <q-list>
-              <q-item data-test="comment-select-edit" clickable v-close-popup @click="editInput(comment.id)">
+              <q-item
+                data-test="comment-select-edit"
+                clickable
+                v-close-popup
+                @click="editInput(comment.id)"
+              >
```

Comment:
Most changed lines do not have an actual change.
It's just your IDE applying different formatting. Please follow `.prettierrc`, use `Volar` and remove `Vetur`. If you need help with these settings let me know.